packaging/linux/run_keybase

```diff
@@ -187,7 +187,7 @@ check_for_url_scheme_launch() {
   # by checking argv.
 
   # If Keybase is already running, then pass it the link and exit.
-  if [[ $# -eq 1 && "$1" =~ ^web+.* ]]; then
+  if [[ $# -eq 1 && "$1" =~ ^(web\+|keybase).* ]]; then
```

Comment:
@eli-schwartz I believe this section is relevant in #18925 as well

Reply:
I'm not sure if this is relevant to the Arch packages because they don't ship this script.